pkg/kubernetes/kube.go

```diff
@@ -70,6 +72,22 @@ const (
 	DefaultFilePath = "templates"
 )
 
+var (
+	// InstanceGroupNamespace is the default namespace to use of instance-group submission
+	InstanceGroupNamespace = "instance-manager"
+	// CustomResourceGroup is the group of the instance-manager API/custom resource definition
+	CustomResourceGroup = "instancemgr"
+	// CustomResourceAPIVersion is the version of the instance-manager API/custom resource definition
+	CustomResourceAPIVersion = "v1alpha1"
+	// CustomeResourceDomain is the domain of the instance-manager API/custom resource definition
+	CustomeResourceDomain = "keikoproj.io"
+	// CustomResourceName is the name of the API/custom resource definition
+	CustomResourceName = fmt.Sprintf("%v.%v", CustomResourceGroup, CustomeResourceDomain)
+	// CustomResourceKind is the resource kind (plural) of the instance-manager API/custom resource definition
+	CustomResourceKind    = "instancegroups"
+	InstanceGroupResource = schema.GroupVersionResource{Group: CustomResourceName, Version: CustomResourceAPIVersion, Resource: CustomResourceKind}
+)
```

Comment:
Can we make some of these consts? Maybe InstanceGroupResource/CustomResourceName no, but can everything else be?

Reply:
done